flycheck.el

```diff
@@ -7662,6 +7663,18 @@ See URL `https://github.com/jimhester/lintr'."
   ;; Don't check ESS files which do not contain R
   :predicate (lambda () (equal ess-language "S")))
 
+(flycheck-define-checker racket
+  "Racket syntax checker.
+
+See URL `https://racket-lang.org/'."
+  :command ("raco" "expand" source-original)
```

Comment:
So it resolves modules correctly and tells if you fuck up a require or something. I figured it was fine since `raco expand` (afaik) shouldn't modify the input source. Is there a nicer way to do this?